apps/theming-designer/src/components/Header.tsx

```diff
@@ -65,64 +87,51 @@ export class Header extends React.Component<IHeaderProps, IHeaderState> {
-              aria-label="Microsoft Fabric Theme Designer page"
-              className={pipeFabricClassName}
-            >
-              <span> | UI Fabric Theme Designer </span>
-            </a>
-          </Stack>
-          <Stack horizontal styles={{ root: { position: 'absolute', right: '25px' } }}>
-            <PrimaryButton text="Export theme" onClick={this.showPanel} styles={{ root: { width: '130px', height: '35px' } }} />
-            <Panel
-              isOpen={this.state.showPanel}
-              type={PanelType.smallFixedFar}
-              onDismiss={this.hidePanel}
-              headerText="Export theme"
-              closeButtonAriaLabel="Close"
-              onRenderFooterContent={this.onRenderFooterContent}
-            >
-              <span>
-                <p>
-                  This code block initializes the theme you have configured above and loads it using the loadTheme utility function. Calling
-                  loadTheme will automatically apply the configured theming to any Fabric controls used within the same app. You can also
-                  export this example to CodePen with a few component examples below.
-                </p>
-              </span>
-              <div className={outputPanelClassName}>
-                <Pivot>
-                  <PivotItem headerText="Code">
-                    <textarea
-                      className={textAreaClassName}
-                      readOnly={true}
-                      spellCheck={false}
-                      value={codeHeader + this.state.themeAsCode}
-                    />
-                  </PivotItem>
-                  <PivotItem headerText="JSON">
-                    <textarea className={textAreaClassName} readOnly={true} spellCheck={false} value={this.state.jsonTheme} />
-                  </PivotItem>
-                  <PivotItem headerText="PowerShell">
-                    <textarea className={textAreaClassName} readOnly={true} spellCheck={false} value={this.state.powershellTheme} />
-                  </PivotItem>
-                </Pivot>
-              </div>
-            </Panel>
-          </Stack>
+      <Stack horizontal verticalAlign="center" grow={0} styles={headerStackStyles}>
+        <Stack horizontal grow={1} verticalAlign="center">
+          <a href="https://www.microsoft.com" title="Microsoft Home Page" aria-label="Microsoft Home Page" className={microsoftLogo}>
+            <img src="https://themingdesigner.blob.core.windows.net/$web/MicrosoftLogo.png" className={microsoftLogo} />
+          </a>
+          <Link
+            href="https://www.aka.ms/themedesigner"
+            title="Microsoft Theme Designer page"
+            aria-label="Microsoft Fabric Theme Designer page"
+            styles={pipeFabricStyles}
+          >
+            | UI Fabric Theme Designer
+          </Link>
         </Stack>
-      </Card>
+        <PrimaryButton text="Export theme" onClick={this.showPanel} />
+        <Panel
```

Comment:
While this is indeed improvement, there are a number of concerns here @aneeshack4 @jdhuntington 

Things like the Panel should be wrapped in their own pure component with minimal input. That way expanding and contracting the panel doesn't re-render the entire panel.

The Panel also has very little to do with the Header, so there is a separation of concerns here. Let the Header be the header, etc.